subgraphs/.folderslintrc

```diff
@@ -0,0 +1,31 @@
+{
+  "comments": "### comments are written like this because of lack of a better way ¯\\_(ツ)_/¯ ###",
+  "rules": [
+    "### graph build outputs and node modules ###",
+    "*/generated/**",
+    "*/build/**",
+    "*/node_modules/**",
+
+    "### generic subgraph structure ###",
+    "*/abis/**",
+    "*/protocols/*/config/deployments/*",
+    "*/protocols/*/config/templates/",
+    "*/protocols/*/src/**",
+    "*/src/**",
+
+    "### uniswap forks structure ###",
+    "uniswap-forks/configurations/*",
+    "uniswap-forks/protocols/*/config/deployments/*",
```

Comment:
Can't you remove these then too since they are redundant?

Reply:
Good catch 💪